libr/asm/asm.c

```diff
@@ -284,9 +284,10 @@ R_API int r_asm_del(RAsm *a, const char *name) {
 R_API int r_asm_is_valid(RAsm *a, const char *name) {
 	RAsmPlugin *h;
 	RListIter *iter;
-	if (!name || !*name) {
-		return false;
-	}
+
+	r_return_val_if_fail (name, false);
+	r_return_val_if_fail (name[0], false);
```

Comment:
cant be name && *name ?

Reply:
again, to exactly know what's wrong in case an error is displayed.